debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/SnapshotReader.java

```diff
@@ -131,14 +133,20 @@ protected void doCleanup() {
      */
     protected void execute() {
         context.configureLoggingContext("snapshot");
-        logger.info("Starting snapshot");
         final AtomicReference<String> sql = new AtomicReference<>();
         final JdbcConnection mysql = context.jdbc();
         final MySqlSchema schema = context.dbSchema();
         final Filters filters = schema.filters();
         final SourceInfo source = context.source();
         final Clock clock = context.clock();
         final long ts = clock.currentTimeInMillis();
+        logger.info("Starting snapshot for {} with user '{}' and pass '{}' at {}", context.connectionString(), mysql.username(),
+                    mysql.password(), new Date(clock.currentTimeInMillis()));
```

Comment:
Do we really want the timestamp in this log, since every log message has the timestamp (if so configured)?